litmus-portal/graphql-server/pkg/handlers/file_handler.go

```diff
@@ -1,4 +1,4 @@
-package file_handlers
+package handlers
```

Comment:
Probably best to rename package to `rest` instead of just handlers

Reply:
what about rest_handlers?

Reply:
That's okay too

Reply:
done